runtime/tests/tests.rs

```diff
@@ -136,6 +136,7 @@ fn start_session(session_index: SessionIndex) {
 	for i in Session::current_index()..up_to_session_index {
 		System::set_block_number((i + 1).into());
 		pallet_babe::CurrentSlot::put(Babe::current_slot() + EpochDuration::get());
+		Timestamp::set_timestamp((System::block_number() * 1000).into());
```

Comment:
👍 what do u think about using `ExpectedBlockTime::get()` (which is configured as 3 seconds) instead of 1000?

Reply:
This change should be good, but when do the replacement,  all the test for `rewards` failed. Not sure what's the problem

Reply:
Yeah they become off by just 1 and I'm not sure why that happens yet.